src/App/App.js

```diff
@@ -2,18 +2,18 @@ import React, { Component } from 'react';
 import './App.css';
 import 'bootstrap/dist/css/bootstrap.min.css';
 import { Container } from 'react-bootstrap'
-import Deck from '../Deck.js';
 import Loader from '../Loader/Loader.js'
 import DailyReading from '../DailyReading/DailyReading.js'
+import NavBar from '../NavBar/NavBar.js'
+import { fetchCards } from '../fetch/fetch-requests';
```

Comment:
Thanks for working on getting the Navbar going, I'm excited!

Reply:
Great executive decision to make the nav bar! Kathryn, you're awesome!